MCFrank16/epic-mail-v2
# #164682442 add the user model and controller

#### What does this PR do?
this pull request merges the changes from the compare branch with the develop

#### Description of Task to be completed?
have the model for inserting a new user in the database
have the controller to check whether the process of inserting a new user has been successful or not.
a little bit of changes on the folder structure
etc

#### How should this be manually tested?
clone the repo and run the server and make sure you have postman to hit the endpoints

#### What are the relevant pivotal tracker stories?
#164682442
